server/routes/functionalSkillsResolver.test.ts

```diff
@@ -209,4 +209,47 @@ describe('functionalSkillsResolver', () => {
       expect(actual).toEqual(expected)
     })
   })
+
+  describe('functionalSkillsByType', () => {
+    it('should return Functional Skills by type given Assessments and type', () => {
+      // Given
+      const assessments = [
+        { type: 'ENGLISH', grade: 'Level 1', assessmentDate: TEN_DAYS_AGO },
+        { type: 'MATHS', grade: 'Level 1', assessmentDate: TEN_DAYS_AGO },
+        { type: 'DIGITAL_LITERACY', grade: 'Level 1', assessmentDate: TEN_DAYS_AGO },
+        { type: 'DIGITAL_LITERACY', grade: 'Level 3', assessmentDate: YESTERDAY },
+        { type: 'MATHS', grade: 'Level 2', assessmentDate: FIVE_DAYS_AGO },
+        { type: 'DIGITAL_LITERACY', grade: 'Level 2', assessmentDate: FIVE_DAYS_AGO },
+        { type: 'MATHS', grade: 'Level 3', assessmentDate: YESTERDAY },
+      ] as Array<Assessment>
+
+      const expected = [{ type: 'ENGLISH', grade: 'Level 1', assessmentDate: TEN_DAYS_AGO }] as Array<Assessment>
+
+      // When
+      const actual = functionalSkillsByType(assessments, 'ENGLISH')
+
+      // Then
+      expect(actual).toEqual(expected)
```

Comment:
Given that you change the implementation to sort by date (see my other comment), then I think it would be worth changing this test so that we are filtering by type `DIGITAL_LITERACY`, because there are 3 of them with various assessment dates. This would then test both the filtering and sorting 👍

Reply:
Makes sense

Reply:
Thats a much better test; perfect ❤️ 
Thanks 👍